docs/branching/index.md

```diff
@@ -2,6 +2,6 @@
 layout: default
 title: Branching Model
 description: At its core, lakeFS uses a Git-like branching model that scales to Petabytes of data by utilizing S3 or GCS for storage.
-nav_order: 3
+nav_order: 25
```

Comment:
why?

Reply:
In order to insert the `Azure deployment` page right after `AWS deployment'  I needed to shift all the other pages by one, So I multiplied them by five making it more convenient for the following cases.